.evergreen/download-mongodb.sh

```diff
@@ -51,7 +51,7 @@ get_mongodb_download_url_for ()
    _DISTRO=$1
    _VERSION=$2
 
-   VERSION_50="5.0.0"
+   VERSION_50="5.0.2"
    VERSION_44="4.4.6"
    VERSION_42="4.2.15"
    VERSION_40="4.0.25"
```

Comment:
Do you mind updating to 4.0.26 and 4.4.8 as well?

Reply:
Done